tests/components/bmw_connected_drive/test_diagnostics.py

```diff
@@ -1,18 +1,17 @@
 """Test BMW diagnostics."""
 import datetime
-import json
 import os
 import time
 
 from freezegun import freeze_time
+from syrupy.assertion import SnapshotAssertion
```

Comment:
Using SnapshotAssertion is great, but it should be a standalone/preliminary PR.

Reply:
Moved SnapshotAssertion changes to #90128.